### PR TITLE
Rename, typo, lint

### PR DESCRIPTION
**Pull request by HazemKhaled**
## :memo: Description

Just some linting

### :gem: Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### :scroll: Example code
```js

``` 

## :vertical_traffic_light: How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## :checkered_flag: Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] **I have added tests that prove my fix is effective or that my feature works**
- [ ] **New and existing unit tests pass locally with my changes**
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/common.ts [hunk 1]
```diff
@@ -6,15 +6,15 @@ export interface I18nString {
 }
 
 export interface Address {
-  first_name: string;
-  last_name: string;
-  company: string;
   address_1: string;
```
Comment (neer14) [inline src/common.ts:9]: address should look like this
https://github.com/Knawat/sp/blob/fb86c9fcb19577419992de9549782d14e7830353/mixins/openapi/suppliers.openapi.mixin.js#L223

File: src/product.ts [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
+  categories: {
+    id: number;
+    name: I18nString;
+  };
+  description: I18nString;
 
   externalId: string;
 
-  supplier: string;
+  externalUrl: string;
```
Comment (neer14) [inline src/product.ts:15]: it should be source_url

File: src/product.ts [hunk 3]
```diff
@@ -1,42 +1,42 @@
...
   variations: ProductVariation[];
 }
 
 interface ProductVariation {
-  sku: string;
-  externalId: string;
+  attributes: ProductVariationAttribute[];
   cost_price: number;
-  sale_price: number;
+  externalId: string;
```
Comment (neer14) [inline src/product.ts:34]: what is externalId?

File: src/supplier.ts [hunk 4]
```diff
@@ -1,88 +1,88 @@
...
-  id: string;
-  project_key: string;
-  logo: string;
-  email: string;
-  sku_prefix: string;
   active: boolean;
+  address: SupplierAddress[];
+  bank: Bank;
   beta: boolean;
+  billing: {
```
Comment (neer14) [inline src/supplier.ts:8]: it should match with https://github.com/Knawat/sp/blob/fb86c9fcb19577419992de9549782d14e7830353/mixins/openapi/suppliers.openapi.mixin.js#L244

File: src/supplier.ts [hunk 5]
```diff
@@ -1,88 +1,88 @@
...
+    data: {
+      access_token: string;
+      fields: unknown;
     };
-  languages: string[];
-  company_description: string;
+    enabled: boolean;
+    type: string;
+  };
   jscrapper_urls: string;
```
Comment (neer14) [inline src/supplier.ts:36]: type should be array

File: src/product.ts (hunk 2 above)
Comment (HazemKhaled) [reply src/product.ts:15]: this object for MP Products

File: src/product.ts (hunk 3 above)
Comment (HazemKhaled) [reply src/product.ts:34]: this object for MP Products

**Pull request merged**